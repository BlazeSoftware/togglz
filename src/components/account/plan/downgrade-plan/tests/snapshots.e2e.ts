import snapIt from '../../../../../test/snap-it';

describe('delete environment', () => {
  const user = {
    email: 'another@dot.com',
  };

  snapIt('account-downgrade-plan')(
    'renders correctly',
    `<account-downgrade-plan user=${user}></account-downgrade-plan>`
  );
});
