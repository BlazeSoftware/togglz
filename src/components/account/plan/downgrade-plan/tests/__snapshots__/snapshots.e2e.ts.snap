// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`delete environment renders correctly 1`] = `
"<account-downgrade-plan user=\\"[object\\" object]=\\"\\" class=\\"hydrated\\"><blaze-drawer class=\\"hydrated\\"><!----><div aria-hidden=\\"true\\" class=\\"c-overlay c-overlay--fullpage\\"></div><aside aria-expanded=\\"async isOpen() {
        return this._isOpen;
    }\\" class=\\"o-drawer o-drawer--right\\"><button type=\\"button\\" class=\\"c-button c-button--close\\">Ã—</button><blaze-card class=\\"hydrated\\"><!----><div class=\\"c-card\\"><form><blaze-card-header class=\\"hydrated\\"><!----><header class=\\"c-card__header\\"><h2 class=\\"c-heading\\">Downgrade to Starter</h2></header></blaze-card-header><blaze-card-body class=\\"hydrated\\"><!----><div class=\\"c-card__body\\"><blaze-alert class=\\"hydrated\\"><!----><div role=\\"alert\\" hidden=\\"\\" class=\\"c-alert u-display-none\\"><div></div><div></div></div></blaze-alert><p class=\\"c-paragraph u-text--loud\\">Are you sure you want to downgrade?</p><p class=\\"c-paragraph\\">No features or environments will be deleted, however your API requests will be limited.</p><p class=\\"c-paragraph u-text--quiet u-small\\">It is recommended you update your site or app appropriately before downgrading.</p></div></blaze-card-body><blaze-card-footer class=\\"hydrated\\"><!----><footer class=\\"c-card__footer\\"><button class=\\"c-button c-button--block c-button--warning\\"><span class=\\"c-button__icon-left\\" aria-hidden=\\"\\"><i aria-hidden=\\"\\" class=\\"fa-fw fas fa-arrow-down\\"></i></span>Switch to Starter plan</button></footer></blaze-card-footer></form></div></blaze-card></aside></blaze-drawer></account-downgrade-plan>"
`;
